packages/core-transaction-pool-mem/lib/defaults.js

```diff
@@ -2,6 +2,11 @@
 
 module.exports = {
   enabled: true,
+  // How often to save/sync the transaction pool to a persistent storage.
+  // The number designates count of added or deleted transactions that are not in
+  // the persistent storage. When that number is reached a new sync of the pool is
+  // triggered.
+  syncToDiskMods: 512,
```

Comment:
I would recommend to rename this to something like `syncInterval` as it will make the comment obsolete because `Interval` indicates that the `sync` will happen periodically at that number.

Reply:
Done in 7d536a3d